.github/renovate.json

```diff
@@ -56,12 +56,12 @@
 			"matchUpdateTypes": ["digest", "patch", "minor", "major"]
 		},
 		{
-			"matchPackagePatterns": [
-				"@insurgent/conventional-changelog-preset",
-				"@insurgent/commitlint-config"
-			],
 			"groupName": "semantic-release related packages",
-			"matchUpdateTypes": ["digest", "patch", "minor", "major"]
+			"matchUpdateTypes": ["digest", "patch", "minor", "major"],
+			"matchPackageNames": [
+				"/@insurgent/conventional-changelog-preset/",
+				"/@insurgent/commitlint-config/"
```

Comment:
The middle `/` might cause an issue, need to check the docs/code to see if that would be the case before merging.

Reply:
I'm trying to validate this according to their docs but it keeps hanging on my machine. maybe you could try it to see if you get different results? https://docs.renovatebot.com/config-validation/

the command I ran was `npx --yes --package renovate -- renovate-config-validator` at the package root

Reply:
Mmh, the config-validator has probably already been run on this PR. but anyway I guess the guys at Renovate are smart enough to have taken this into account :sweat_smile: that comment was pretty useless in hindsight